examples/app-router/app/api/after/ssg/route.tsx

```diff
@@ -0,0 +1,13 @@
+import { unstable_cache } from "next/cache";
+import { NextResponse } from "next/server";
+
+export const dynamic = "force-static";
+
+export async function GET() {
+  const dateFn = unstable_cache(() => new Date().toISOString(), ["date"], {
+    tags: ["date"],
+  });
+  const date = await dateFn();
+  console.log("date", date);
+  return NextResponse.json({ date: date });
```

Comment:
shouldn't this be a `*.ts` file since its a route?

Reply:
Good point, i'll change it

Reply:
great work on this PR btw